pkg/swiveltesting/dep.go

```diff
@@ -86,10 +96,12 @@ func Deploy(e *Env) (*Dep, error) {
 		SigFake:              sigContract,
 		HashFakeAddress:      hashAddress,
 		HashFake:             hashContract,
-		Erc20Address:         ercAddress,
 		Erc20:                ercContract,
-		CompoundTokenAddress: ctAddress,
+		Erc20Address:         ercAddress,
 		CompoundToken:        ctContract,
+		CompoundTokenAddress: ctAddress,
+		Erc4626:              tvContract,
+		Erc4626Address:       tvAddress,
```

Comment:
```suggestion
		Erc4626:              erc4626Contract,
		Erc4626Address:       erc4626Address,
```
See comment above about matching pattern.